src/subscript/runrms/runrms.py

```diff
@@ -339,25 +342,28 @@ def requested_rms_version(self):
             raise RuntimeError("Executable is not found, probably a config/setup error")
 
         pypath = self.setup[rmssection][proposed_version].get("pythonpath", None)
-        self.pythonpath = self._process_pypath(pypath)
+        self.pythonpath = self._process_genericpath(pypath)
 
         # test pythonpath
         pypathtest = self.setup[rmssection][proposed_version].get(
             "pythonpathtest", None
         )
-
         if pypathtest:
-            self.testpythonpath = self._process_pypath(pypathtest)
+            self.testpythonpath = self._process_genericpath(pypathtest)
+
+        pluginspath = self.setup[rmssection][proposed_version].get("pluginspath", None)
+        if pluginspath:
+            self.pluginspath = self._process_genericpath(pluginspath)
 
         self.tcltkpath = self.setup[rmssection][proposed_version].get("tcltkpath", None)
 
         logger.debug("EXECUTABLE: %s", self.exe)
         logger.debug("PYTHONPATH: %s", self.pythonpath)
+        logger.debug("PLUGINSPATH: %s", self.pluginspath)
         logger.debug("TCLTK: %s", self.tcltkpath)
 
-    def _process_pypath(self, pypath):
-        """
-        The proposed pythonpath or testpythonpath from setup is a list.
+    def _process_genericpath(self, pypath):
```

Comment:
Function name and docstring are not fully compatible, is this a function getting something or a function processing something?

Is it perhaps a valdation function?